package/aws/securitygroup/definition.yaml

```diff
@@ -0,0 +1,488 @@
+apiVersion: apiextensions.crossplane.io/v1
+kind: CompositeResourceDefinition
+metadata:
+  name: xawssecuritygroups.ec2.crossplane.dfds.cloud
+spec:
+  defaultCompositionRef:
+    name: awssecuritygroups.ec2.crossplane.dfds.cloud
+  group: ec2.crossplane.dfds.cloud
```

Comment:
The intention was that these group names are generic and not cloud/service specific so that any azure or gcp instances slot into the same groups in future. Should perhaps be 'compute' or 'network' which i think is more suitable for a security group even if it does not sit under there for AWS

Reply:
Thanks for clarifying. That makes sense :)
The corresponding resource on Azure is called: Network Security Groups
Looks like the closest thing on Google Cloud is firewall rules. So I suggest we use the 'network' sub name.